### PR TITLE
Update to latest stable Roslyn packages

### PR DESCRIPTION
**Pull request by nguerrera**
@michaelcfanning 

The build was broken in VS 2015 Update 1 RTM (analyzer exception) without this.

### CODE REVIEW

File: src/BinSkim.Rules/BinSkim.Rules.csproj [hunk 1]
```diff
@@ -25,6 +25,10 @@
       <HintPath>..\packages\Driver.Utilities.1.0.0\lib\net40\Driver.Utilities.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.CodeAnalysis.Common.1.1.1\lib\net45\Microsoft.CodeAnalysis.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```
Comment (nguerrera) [inline src/BinSkim.Rules/BinSkim.Rules.csproj]: Looks like I may have clicked some wrong buttons and added refs too all projects.

**Comment by nguerrera** [comment]
Don't merge. Fixing.

**Comment by nguerrera** [comment]
Fixed. :)

**Comment by michaelcfanning** [comment]
Looks good to me.

**Pull request merged**